packages/workflows-engine-inmemory/src/repositories/workflow-execution.ts

```diff
@@ -0,0 +1,23 @@
+import { Context } from "@medusajs/types"
+import { DALUtils } from "@medusajs/utils"
+import { WorkflowOrchestratorTypes } from "@medusajs/workflows-sdk"
+import { SqlEntityManager } from "@mikro-orm/postgresql"
+import { WorkflowExecution } from "@models"
+
+// eslint-disable-next-line max-len
+export class WorkflowExecutionRepository extends DALUtils.mikroOrmBaseRepositoryFactory(
+  WorkflowExecution
+) {
+  async upsert(
```

Comment:
comment: I believe we have `upsert` support now with https://github.com/medusajs/medusa/pull/6127 merged